tests/spec/fs.mknod.spec.js

```diff
@@ -1,82 +1,94 @@
 var util = require('../lib/test-utils.js');
 var expect = require('chai').expect;
 
-describe('fs.mknod', function() {
+describe('fs.mknod', function () {
```

Comment:
Let's consider cleaning up the white spaces in a separate PR! We should aim to keep PR's confined to one specific thing - in this case, it's simply adding a new test as stated in #483. Another issue should be opened with a title of `Remove whitespace in test files` if you'd like!

With that being said, can you please revert the white space changes in a new commit? Please let me know if you need any help with this! 😁